### DISCUSSION TITLE
loco anchors  don't show on the client

### DISCUSSION THREAD

**Discussion by nfu64967512** [bitcraze/discussions#1046] [Q&A] [answered]
Hi! I have some questions for loco position ,I have set up for all anchors and apply them number. But when I open client and  connect crazyflie 2.0 then turn to loco position page, I see all anchors status are green, but when I try to  use "configure position" and click "get form anchors", nothing happen on the window, how should I figure out this?
Also, there are no anchors on my 3D window, and number 0 is usually not working.
Thank you for solving my question! 
![client](https://github.com/bitcraze/discussions/assets/148971090/2a6ca39e-4753-4a82-8365-f47107ee245c)
![客戶端](https://github.com/bitcraze/discussions/assets/148971090/84cbc5b2-7eb7-41e2-b428-dd242648d5d9)

**Comment by krichardsson** [comment, marked as answer]
I the anchors are new, they do not yet have any position stored in them and this is why they do not show up. 
Just use the "Add anchor" button to configure your anchors and then click "Write to anchors" to write the position to the anchors.

The positions should now be stored in the anchors and they will start to transmit it as a part of their packets and when the data propagates up through the CF and to the client, they should show up in the client.

**Reply by nfu64967512** [reply]
Hi! sorry for late response. I'll try it later.
 If it works I'll leave comment and close this Q&A.

**Reply by nfu64967512** [reply]
OK I try your suggestion, add eight anchors then write their X,Y,Z and try to let crazyflie stay in the middle and it success shows on my configure window . Then I push take off the drone will fly to other side instead of straight take off. But it will come back to loco position's valid range. My crazyflie has flow deck so I don't know it will influence it or not, but it seems like still unstable for now. Do you have any suggestion for me?
Thank you for helping!

**Reply by krichardsson** [reply]
Please start a new discussion for this as it is a different problem

**Reply by nfu64967512** [reply]
Ok, I'll try it later! Thanks!
